extensions/EXT/EXT_depth_clamp.txt

```diff
@@ -0,0 +1,222 @@
+      the depth clamping is then not applied.
+
+    What push/pop attrib bits should affect the depth clamp enable?
+
+      RESOLUTION:  GL_ENABLE_BIT and GL_TRANSFORM_BIT.
+
+    How does depth clamping interact with depth replace operations (say
+    from NV_texture_shader)?
+
+      RESOLUTION:  The depth clamp operation occurs as part of the depth
+      test so depth clamping occurs AFTER any depth replace operation
+      in the pipeline.  A depth replace operation can reassign the
+      fragment's zw, but depth clamping if enabled will subsequently
+      clamp this new zw.
+
+    Does depth clamping affect read/draw/copy pixels operations involving
+    depth component pixels?
+
+      RESOLUTION:  No.
+
+    Does depth clamping occur after polygon offset?
+
+      RESOLUTION:  Yes.  Depth clamping occurs immediately before the
+      depth test.
+
+    Can fragments with w_c <=0 be generated when this extension is supported?
+
+      RESOLUTION: No. The OpenGL ES Specification clarifies that only
+      primitives that lie in the region w_c > 0 need to be produced by
+      clipping and consequently such fragments should also never be generated
+      when this extension is supported. 
+
+New Procedures and Functions
+
+    None
+
+New Tokens
+
+    Accepted by the <cap> parameter of Enable, Disable, and IsEnabled,
+    and by the <pname> parameter of GetBooleanv, GetIntegerv,
+    GetFloatv, and GetDoublev:
+
+        DEPTH_CLAMP_EXT                               0x864F
+
+Additions to Chapter 12 of the OpenGL ES 3.2 Specification (Fixed-Function Vertex Post-Processing)
+
+ --  Section 12.5 "Primitive Clipping"
+
+    Add to the end of the 1st paragraph:
+
+    "Depth clamping is enabled with the generic Enable command and
+    disabled with the Disable command. The value of the argument to
+    either command is DEPTH_CLAMP. If depth clamping is enabled, the
+    "-w_c <= z_c <= w_c" plane equation are ignored by view volume
+    clipping (effectively, there is no near or far plane clipping)."
+
+Additions to Chapter 15 of the OpenGL ES 3.2 Specification (Writing Fragments and Samples to the Framebuffer)
+
+ --  Section 15.1.3 "Depth buffer test"
+
+    Add to the end of the 2nd paragraph:
+
+    "If depth clamping (see section 12.15) is enabled, before the
+    incoming fragment's z_w is compared z_w is clamped to the range
+    [min(n,f),max(n,f)], where n and f are the current near and far
+    depth range values (see section 12.6.1)."
+
+Additions to the AGL/GLX/WGL Specifications
+
+    None
+
+GLX Protocol
+
+    None
+
+Errors
+
+    None
+
+New State
+
+Add to table 6.4, transformation state
+
+Get Value       Type  Get Command  Initial Value  Description     Sec      Attribute
+--------------  ----  -----------  -------------  --------------  ------   ----------------
+DEPTH_CLAMP     B     IsEnabled    False          Depth clamping  12.5   transform/enable
+                                                  on/off
```

Comment:
DEPTH_CLAMP_EXT here as well. 

You should also remove the Attribute column as that is not applicable in ES.

Reply:
done